r3bdata/DataLinkDef.h

```diff
@@ -66,9 +66,14 @@
 #pragma link C++ class R3BRpcPmtMappedData+;
 #pragma link C++ class R3BRpcRefMappedData+;
 #pragma link C++ class R3BRpcStripMappedData+;
-#pragma link C++ class R3BRpcCalData+;
-#pragma link C++ class R3BRpcHitData+;
+#pragma link C++ class R3BRpcPmtPreCalData+;
```

Comment:
What is the difference between Pmt and strip? 
I guess both signals are coming from the rpc electronics, same board, same calibrations? 
If needed, it could be better the use of R3BRpcPaddlePreCalData with a data member pmt.

Reply:
Even though the electronics are the same, the calibration and analysis, after pre-cal, will be different. I discussed this with Gabriel and we decided that separating them would make it easier to work. If it really is required to change I will do it, but we are rather close to the test beam.